### PR TITLE
Using artillery.io to generate traffic

### PR DESCRIPTION
**Pull request by faisaladnanpeltops**
Hi, here is my submission to generate traffic using artillery.io

### CODE REVIEW

**Review by yardenshoham — APPROVED** [review]

File: artillery/HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Deploy artillery
+```
+    kubectl apply -f artillery.yaml
+```
+# Customising Test Flow
+## Edit load.yml
+## Build docker file
+```
+    docker build -t <DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>:<VERSION> -f Dockerfile 
+```
```
Comment (dinis-cruz-gw) [inline artillery/HOWTO.md:25]: I don’t think we need the docker hub references for now

File: artillery/HOWTO.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+## Push image to Docker Hub
+```
+    docker login --username=<DOCKER_HUB_ACCOUNT>
+    docker push <DOCKER_HUB_ACCOUNT>/<IMAGE_NAME>:<VERSION>
+```
```
Comment (dinis-cruz-gw) [inline artillery/HOWTO.md:30]: Same here, this is not needed

File: artillery/load.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    - get:
+        url: "/technology"    
+    - get:
+        url: "/products"
+    - get:
+        url: "/pricing"
+    - get:
+        url: "/resources"
+    - get:
+        url: "/company"
```
Comment (dinis-cruz-gw) [inline artillery/load.yml:20]: Cool, where is the output stored?

File: artillery/HOWTO.md (hunk 1 above)
Comment (dinis-cruz-gw) [inline artillery/HOWTO.md:25]: Can you add more info to this README about what artilary is doing

Can you also create a diagram that shows what is going on

File: artillery/HOWTO.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- Open https://glasswallsolutions.com/resources
+- Open https://glasswallsolutions.com/company
+## Install minikube
+```
+    brew install minikube
+```
+## Start minikube
+```
+    minikube start --driver=virtualbox
+```
```
Comment (dinis-cruz-gw) [inline artillery/HOWTO.md:15]: Nice touch :) 

(The installation and setup of minikube)

File: artillery/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,8 @@
+FROM node:12
```
Comment (dinis-cruz-gw) [inline artillery/Dockerfile:1]: As mentioned in the slack environment, please refactor these files to be under a folder with your name

**Review by dinis-cruz-gw — CHANGES_REQUESTED** [review]

File: artillery/HOWTO.md (hunk 1 above)
Comment (faisaladnanpeltops) [reply artillery/HOWTO.md]: I'll remove this step and keep the howto simple.

File: artillery/load.yml (hunk 3 above)
Comment (faisaladnanpeltops) [reply artillery/load.yml]: The output will be stored in statsd and be visualised in graphite.

File: artillery/HOWTO.md (hunk 1 above)
Comment (faisaladnanpeltops) [reply artillery/HOWTO.md]: Sure thing

File: artillery/HOWTO.md (hunk 4 above)
Comment (faisaladnanpeltops) [reply artillery/HOWTO.md]: Thank you :)

File: artillery/Dockerfile (hunk 5 above)
Comment (faisaladnanpeltops) [reply artillery/Dockerfile]: Ok.

**Review by faisaladnanpeltops — COMMENTED** [review]
I've read the requested changes and updating them in the forked repo.

File: upwork-devs/faisal-adnan/artillery.yaml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: artillery
+  labels:
+    env: test
+spec:
+  containers:
+  - name: gsl-artillery
+    image: peltops/gsl_artillery:v2
```
Comment (DinisCruz) [inline upwork-devs/faisal-adnan/artillery.yaml:10]: can you document the command to build this image

**Pull request merged**